docs/user-guide/cinder.md

```diff
@@ -0,0 +1,1351 @@
+- [3. Prepare transport protocol requirements](#3-prepare-transport-protocol-requirements)
+- [4. Setting initial defaults](#4-setting-initial-defaults)
+- [5. Configuring the API service](#5-configuring-the-api-service)
+- [6. Configuring the scheduler service](#6-configuring-the-scheduler-service)
+- [7. Configuring the volume service](#7-configuring-the-volume-service)
+- [8. Configuring the backup service](#8-configuring-the-backup-service)
+- [9. Automatic database cleanup](#9-automatic-database-cleanup)
+- [10. Preserving jobs](#10-preserving-jobs)
+- [11. Resolving hostname conflicts](#11-resolving-hostname-conflicts)
+
+
+## 1. Terminology
+
+There are some terminology that is important to clarify when discussing cinder:
+
+* Storage back-end: Refers to a physical Storage System where the data for the
+  *volumes* is actually stored.
+* Cinder driver: Is the code in cinder that knows how to talk to a Storage
+  back-end. Configured using the `volume_driver` option.
+* Cinder back-end: Refers to a logical representation of the grouping of a
+  cinder driver with its configuration. This grouping is used to manage and
+  address the volumes present in a specific Storage back-end. The name of this
+  logical construct is defined using the `volume_backend_name` option.
+* Storage pool: Logical grouping of volumes in a given Storage back-end.
+* Cinder pool: The representation in Cinder of a Storage pool.
+* Volume host: This usually refers to the way cinder uses to address volumes.
+  There are two different representations, short `<hostname>@<backend-name>` and
+  full `<hostname>@<backend-name>#<pool-name>`.
+
+## 2. Changes from older releases
+
+If you are familiar with previous Red Hat OpenStack Platform releases that were
+deployed using TripleO, you will notice enhancements and improvements as well as
+some differences such as:
+
+* Multiple volume back-ends:
+  * Very easy to deploy.
+  * Adding back-ends is fast and doesn’t affect other running back-ends.
+  * Removing back-ends is fast and doesn’t affect other running back-ends.
+  * Making configuration changes to one back-end doesn’t affect other back-ends.
+  * Each back-end can use its own vendor specific container image. No more
+    building custom images to hold dependencies from two drivers.
+* Recommended deployment avoids inconsistent scheduling issues.
+* No more pacemaker. OpenShift functionality has been leveraged to not need it.
+* Easier way to debug service code when faced with difficult to resolve issues.
+
+
+## 3. Prepare transport protocol requirements
+
+The OpenStack control plane services that use volumes, such as cinder volume and
+cinder backup, require the OpenShift cluster’s support for them, as it needs to
+run some daemons and kernel modules, such as `iscsid` and `multipathd`.
+
+These daemon and kernel modules must be available in all the OpenShift nodes
+where the cinder volume and backup services could run, not only on the ones
```

Comment:
I don't mention anything about glance, since I consider that is a glance documentation matter.

We can always add it in the future if glance doesn't.